packages/serverpod/lib/src/server/websocket_request_handlers/endpoint_websocket_request_handler.dart

```diff
@@ -65,55 +65,50 @@ abstract class EndpointWebsocketRequestHandler {
           var endpointName = data['endpoint'] as String;
           var serialization = data['object'] as Map<String, dynamic>;
 
-          var endpointConnector =
-              server.endpoints.getConnectorByName(endpointName);
-          if (endpointConnector == null) {
+          EndpointConnector endpointConnector;
```

Comment:
FYI: There is also authentication happening in the `_callStreamOpened` method. Do we need to think about that?

Reply:
Oh, good catch! Will refactor to cover that path too 👍

Reply:
[0493f37](https://github.com/serverpod/serverpod/pull/2676/commits/0493f37f73cd42d3f3666bfa9630ff82ad3750af)

Reply:
Yes, exactly!